gapis/service/service.proto

```diff
@@ -313,6 +313,48 @@ message FindResponse {
   }
 }
 
+
+enum ClientAction {
+  Undefined = 0;
+  Show = 1;
+  Hide = 2;
+  MouseClickLeft = 3;
+  MouseClickMiddle = 4;
+  MouseClickRight = 5;
+  MouseDoubleClick = 6;
+  MouseDrag = 7;
+  MouseScroll = 8;
+  MouseEnter = 9;
+  MouseExit = 10;
+  KeyPressUpOrDown = 11;
+  KeyPressLeftOrRight = 12;
+}
+
+message ClientInteraction {
+  string view = 1;
+  string control = 2;
+  ClientAction action = 3;
```

Comment:
The implementation just converts this to a string... so why not just make it a string in the API?

Reply:
It's discouraged to expose string fields where unnecessary. This greatly increases the possibility of accidentally leaking PII. The couple of strings I have already are questionable, and probably should be turned into some sort of enum if we can agree on them.